### ISSUE TITLE
Normalize `Table`

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Stdlib#141]
### Is your feature request related to a problem?

Normalization mean to shift and squeeze the data into the interval [0, 1]. More generally, we can do that for any interval [minimum, maximum].

### Desired solution

* Add a class `RangeScaler` in `safeds.data.tabular.transformation` that wraps the [`MinMaxScaler` of `scikit-learn`](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#sklearn.preprocessing.MinMaxScaler)
* Make it a subclass of `InvertibleTableTransformer`
* Add two parameters to the `__init__` method to set the minimum (float, default 0.0) and maximum (float, default 1.0).
* Raise a value error if minimum >= maximum.
